src/heitech.zer0mqXt.core/Main/Zer0Mq.cs

```diff
@@ -0,0 +1,44 @@
+using heitech.zer0mqXt.core.Adapters;
+using heitech.zer0mqXt.core.infrastructure;
+
+namespace heitech.zer0mqXt.core.Main
+{
+    public class Zer0Mq : IZer0MqBuilder
+    {
+        private ILogger _logger;
+        private ISerializerAdapter _serializer;
+        private Zer0Mq()
+        {
+            _logger = new BasicLogger();
```

Comment:
We create here two instances of `BasicLogger` (`InternalAdapters` creates the 2nd instance).
Is this OK to have two instances `BasicLogger`? Or should `InternalAdapters` have another c'tor to reuse that `BasicLogger`?

Reply:
No the internaladapter is flawed in this PR. I first tried to merge ISerializerAdapter and Ilogger by implementing it on the InternalAdapter. Yet the Logger is already an interface and fine as is, so this is just a rudimentary field which I am going to remove.